distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc

```diff
@@ -61,4 +61,12 @@ These are the attributes common to any of the media file types which support add
 * `contact.creator-name`
 * `contact.publisher-name`
 * `contact.point-of-contact-phone`
-* `topic.keyword`
\ No newline at end of file
+* `topic.keyword`
+
+Some file types support attributes unique to that individual type:
+
+.Additional Unique Attributes by File Type
+Mp4:: `ext.mp4.audio-sample-rate`
```

Comment:
Is this an example or something? The way it flows with the rest of the document makes it look odd. Maybe it should be segregated from the rest of the document more clearly?